### ISSUE TITLE
ODF2PDFViaITextConverter only uses first page header and standard header was ignored and replaced by the first page header 

### ISSUE THREAD

**Issue by GoogleCodeExporter** [dbarra/xdocreport#124]
```
What steps will reproduce the problem?
1.InputStream in = new FileInputStream(new File("sample.odt"));
2.Options 
options=Options.getTo(ConverterTypeTo.PDF).via(ConverterTypeVia.ITEXT);
3. OutputStream out = new FileOutputStream(new File("sample.pdf"));
4. ODF2PDFViaITextConverter.getInstance().convert(in, out, options);

What is the expected output? What do you see instead?
As the sample.odt has first page header different from subsequent page header 
the the resulting pdf file should have two different headers but, on the pdt 
file produced, all the pages have the same header, which is the first page 
header.

What version of the product are you using? On what operating system?
ODFDOM v.0.88 | xdocreport v. 0.97 | Linux mint 13 (Ubuntu 12.04)

Please provide any additional information below.
Same situation occurs with report.convert(context, options, out); when context 
is based on an odt template with two different headers

```

Original issue reported on code.google.com by `GeorgeWa...@gmail.com` on 23 Jun 2012 at 3:27
- Merged into: #102

**Comment by GoogleCodeExporter** [comment]
```
Hi George,

Could you attach your odt please.
Many thank's

@Leszek:could you see the problem please if you have time. Many Than's

Regards Angelo
```

Original comment by `angelo.z...@gmail.com` on 23 Jun 2012 at 3:46
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
George

I've fixed similar problem in 0.9.8 version.
See http://code.google.com/p/xdocreport/issues/detail?id=102
However 0.9.8 is not officially released yet.
Could you try your example with 0.9.8 SNAPSHOT?
If not please provide your *.odt example. I could look at it.
Regards
Leszek
```

Original comment by `abe...@gmail.com` on 25 Jun 2012 at 8:12

**Comment by GoogleCodeExporter** [comment]
Original comment by `angelo.z...@gmail.com` on 27 Jun 2012 at 11:15

**Comment by GoogleCodeExporter** [comment]
```
Since George did not provided any new material to this bug
I consider this bug as a duplicate of issue 102 and close it.
Issue 102 was fixed in 0.9.8 version
```

Original comment by `abe...@gmail.com` on 7 Aug 2012 at 11:28
- Changed state: **Duplicate**
